### PR TITLE
docs: update documentation for TS models

### PR DESCRIPTION
**Pull request by callicles**
* Updates documentation to remove the mention of prisma models
* Updates the navbar to be collapsed as a default

### CODE REVIEW

File: apps/framework-docs/src/pages/building/data-models/model-data.mdx [hunk 1]
```diff
@@ -2,54 +2,63 @@ import { Cards, Card, Tabs } from "nextra/components";
...
 }
 ```
 
 ## Data Model conventions
 
-As you can see, the data model is very simple. It's just a list of fields and their types. MooseJS's data modeling is built on top of Prisma's intuitive schema modeling language. You can read more about Prisma's schema modeling language [here](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema). We don't currently support all of Prisma's schema modeling features, but we're working on it; and we’ll also soon be adding support to define Data Models as Typescript interfaces.
+As you can see, the data model is very simple. It's just a list of fields and their types.
+MooseJS's data modeling is integrated with the data modeling capabilities of your language of choice.
+For now that's Typescript Interfaces but Python is in the works, leveraging Data classes.
+We don't currently support all of Typescript schema modeling features, but we're working on it.
```
Comment (03cranec) [inline apps/framework-docs/src/pages/building/data-models/model-data.mdx:24]: I would add an explanation about the Key functionality here

Comment (callicles) [reply apps/framework-docs/src/pages/building/data-models/model-data.mdx:24]: The `Key` as in `type Key<T extends string | number> = T;` or key functionality of ts data model in general?

Comment (03cranec) [reply apps/framework-docs/src/pages/building/data-models/model-data.mdx:24]: maybe both? i would explain why that specific line of code is needed / what it does, and then I would explain what marking a property as a Key does. and maybe any limitations on the convention (like, is it 1 and only 1 property must be marked as Key? can the Key only be a string or number type?)

Comment (callicles) [reply apps/framework-docs/src/pages/building/data-models/model-data.mdx:24]: Cool - I believe I addressed the feedback :) Thank you

**Review by 03cranec — APPROVED** [review]

**Pull request merged**